services/concepts/failover-protection.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Learn about failover protection
+sidebar_position: 1
+---
+
+# Failover protection
+
+:::info
+
+Failover support is available on Mainnet only.
```

Comment:
This is currently incorrect and misleading so should be updated before merging. We should list the correct networks that support failover. Additionally, perhaps move the "Partners and privacy policies" content to the "services/reference/[NETWORK]/index.md file for the appropriate networks.

Note: Networks that support failover have different partners so it cannot be generic.